pkg/asset/agent/common.go

```diff
@@ -56,3 +64,57 @@ func IsSupportedPlatform(platform hiveext.PlatformType) bool {
 	}
 	return false
 }
+
+// OCexecute will execute an oc command.
+func OCexecute(pullSecret, command string) (string, error) {
```

Comment:
nit: given that naming is usually difficult, my preference in this case would be for something like `RunOC` or `ExecuteOC`